test/Core.Test/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteClaimedOrganizationUserAccountValidatorTests.cs

```diff
@@ -0,0 +1,386 @@
+﻿using Bit.Core.AdminConsole.Errors;
+using Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers;
+using Bit.Core.AdminConsole.Repositories;
+using Bit.Core.AdminConsole.Shared.Validation;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Repositories;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.AdminConsole.Validators;
+
+[SutProviderCustomize]
+public class DeleteClaimedOrganizationUserAccountValidatorTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task ValidateAsync_ShouldExecuteSyncValidatorsFirst(
+        Guid organizationId,
+        SutProvider<DeleteClaimedOrganizationUserAccountValidator> sutProvider)
+    {
+        // Arrange
+        var request = new DeleteUserValidationRequest
```

Comment:
Hey @eliykat , since I'm going on leave, I'm going to try my best to update the tests and address comments that aren’t related to the command result pattern. I can’t address the command result changes until we align as a team on how we want to move forward, so you’ll probably need to update the tests, but hopefully it’ll just be small typing issues.

Reply:
Sounds good to me!